templates/zerver/emails/email_base.html

```diff
@@ -1,17 +1,350 @@
+        }
+        /* This should also be a block element, so that it will fill 100% of the .container */
+
+        .content {
+            position: relative;
+            box-sizing: border-box;
+            display: block;
+            margin: 20px auto 0 auto;
+            max-width: 580px;
+            padding: 10px;
+        }
+        /* -------------------------------------
+        HEADER, FOOTER, MAIN
+        ------------------------------------- */
+
+        .main {
+            background: #fff;
+            border-radius: 3px;
+            width: 100%;
+        }
+        .wrapper {
+            box-sizing: border-box;
+            padding: 75px 20px 20px 20px;
+        }
+        .footer {
+            clear: both;
+            padding-top: 10px;
+            text-align: center;
+            width: 100%;
+            margin-top: 20px;
+        }
+        .footer td,
+        .footer p,
+        .footer span,
+        .footer a {
+            color: #999999;
+            font-size: 12px;
+            text-align: center;
+        }
+        /* -------------------------------------
+        TYPOGRAPHY
+        ------------------------------------- */
+
+        h1,
+        h2,
+        h3,
+        h4 {
+            color: #000000;
+            font-family: sans-serif;
+            font-weight: 400;
+            line-height: 1.4;
+            margin: 0;
+            margin-bottom: 30px;
+        }
+        h1 {
+            font-size: 35px;
+            font-weight: 300;
+            text-align: center;
+            text-transform: capitalize;
+        }
+        p,
+        ul,
+        ol {
+            font-family: sans-serif;
+            font-size: 14px;
+            font-weight: normal;
+            margin: 10px 0px;
+        }
+        p li,
+        ul li,
+        ol li {
+            list-style-position: inside;
+            margin-left: 5px;
+        }
+        a {
+            color: #45a98e;
+            text-decoration: underline;
+        }
+
+        a:hover {
+            color: #51c6a6;
+        }
+
+        .important {
+            border-bottom: 3px solid #d5e4e9;
+        }
+        /* -------------------------------------
+        BUTTONS
+        ------------------------------------- */
+
+        button {
+            display: block;
+            padding: 10px 0px;
+            margin: 20px auto;
+            width: 200px;
+            border: 1px solid #7fd1ba;
+            background-color: #7fd1ba;
+            border-radius: 4px;
+            font-size: 1rem;
+            outline: none;
+            color: #ffffff;
+            font-family: sans-serif;
+        }
+
+        button a {
+            color: inherit;
+            text-decoration: none;
+            display: block;
+        }
+
+        button a:hover {
+            text-decoration: none;
+            color: #ccf7eb;
+        }
+
+        /* -------------------------------------
+        OTHER STYLES THAT MIGHT BE USEFUL
```

Comment:
i ported all of Brock's changes without refactoring-- will remove the ones that are unnecessary for now.